pcsx2/DEV9/ATA/ATA_Info.cpp

```diff
@@ -254,21 +257,21 @@ void ATA::CreateHDDinfo(u64 sizeSectors)
 	// clang-format off
 	WriteUInt16(identifyData, &index, static_cast<u16>(
 		/*(1 << 8) |						//SET MAX */
-		((lba48Supported ? 1 : 0) << 10) |	//Fixed
+		(lba48Supported << 10) |			//Fixed
 		(1 << 12) |							//Fixed
 		(1 << 13)));						//Fixed      //word 86
 	//Command set/feature enabled/supported (See word 84)
-	WriteUInt16(identifyData, &index, static_cast<u16>((1 << 14) | (1 << 1) | 1));
-	WriteUInt16(identifyData, &index, static_cast<u16>(
-		(1) |								//Fixed
-		((1) << 1)));						//Fixed      //word 87
 	// clang-format on
+	WriteUInt16(identifyData, &index, ( //word 87
+										  (1) | // Fixed
+										  (1 << 1) | // Fixed
+										  (1 << 14))); // Fixed
```

Comment:
I'm personally not a fan of how clang-format has formatted this.
Open to suggestions on how people think this should be formatted